corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js

```diff
@@ -498,32 +502,21 @@ hqDefine("cloudcare/js/formplayer/menus/views", function () {
             }
         },
 
-        addAddressPin: function (geocoder, addressMap, headers, model, addressIndex) {
-            const coordinates = model.attributes.data[addressIndex];
-            if (coordinates) {
-                let latLng = coordinates.split(" ").slice(0,2);
-                if (latLng.length > 1) {
-                    let popupText = "";
-                    headers.forEach((header, index) => {
-                        if (header) {
-                            const valueSanitized = DOMPurify.sanitize(model.attributes.data[index]);
-                            const headerSanitized = DOMPurify.sanitize(header);
-                            popupText += "<b>" + headerSanitized + ":</b> " + valueSanitized + "<br>";
-                        }
-                    });
-                    L.marker(latLng)
-                        .addTo(addressMap)
-                        .bindPopup(popupText);
-
-                    return latLng;
-                }
-            }
-            return null;
+        fontAwesomeIcon: function (iconName) {
+            return L.divIcon({
+                html: `<i class='fa ${iconName} fa-4x'></i>`,
+                iconSize: [12, 12],
+                className: 'myDivIcon'
+            });
         },
 
         loadMap: function () {
             const token = initialPageData.get("mapbox_access_token");
 
+            const locationIcon = this.fontAwesomeIcon("fa-map-marker");
```

Comment:
Should these go inside the try block? They reference `L`, and I'm conscious of the error we saw on staging with `L` not being defined.

Reply:
@orangejenny Do you think the whole loadMap function should go in a try block then? If L cannot be resolved nothing on the map will work.

Reply:
Yeah, I think so. You could just check if L exists and return early if it doesn't.

Reply:
I just realized that I had a big old try catch around most of the code in loadMap. So I just moved the icons in there too